Doc changes for addition of dashboard in windows app. Correction in l… (#2850)
SumoLogic/sumologic-documentation: docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md

sumo.datasource=linux host.name=* metric=system.cpu.utilization state=(user OR system OR wait OR steal OR softirq OR interrupt OR nice) | sum by host.name | outlier
```

> ## Linux App Dashboards

Reviewer: ```suggestion
## Linux app dashboards
```

### Linux - Overview

> See an overview of Linux activity, including the distribution of system events across hosts, group assignment changes, a breakdown of successful and failed logins, sudo attempts, and the count of reporting hosts. Along with this dashboard gives you an at-a-glance view of the key metrics like CPU, memory, disk, network, and TCP connections of all your hosts. You can drill down from this dashboard to the Host Metrics - CPU/Disk/Memory/Network/TCP dashboard by using the honeycombs or line charts in all the panels.

Reviewer: ```suggestion
The **Linux - Overview** dashboard provides an overview of Linux activity, including the distribution of system events across hosts, group assignment changes, successful and failed logins, sudo attempts, and the count of reporting hosts. This dashboard also gives you an at-a-glance view of the key metrics like CPU, memory, disk, network, and TCP connections of all your hosts. You can drill down from this dashboard to the Host Metrics - CPU/Disk/Memory/Network/TCP dashboard by using the honeycombs or line charts in all the panels.
```

<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Host-Metrics-TCP.png' alt="Host Metrics - TCP" />

### The Process Metrics - Overview

The **Process Metrics - Overview** dashboard gives you an at-a-glance view of all the processes by open file descriptors,  CPU usage, memory usage, disk read/write operations and thread count.

> User this dashboard to :

Reviewer: ```suggestion
User this dashboard to:
```

> - Process wise distribution of CPU and memory usage

Reviewer: ```suggestion
- Process wise distribution of CPU and memory usage.
```

> - Process wise read/write operations

Reviewer: ```suggestion
- Process wise read/write operations.
```

<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Process-Metrics-Details.png' alt="Process Metrics - Details" />

### Linux - Event Sources

> See information about system events, including their distribution across hosts, event counts per host by hour, and even counts by host and service.

Reviewer: ```suggestion
The **Linux - Event Sources** dashboard provides information about system events, including their distribution across hosts, event counts per host by hour, and even counts by host and service.
```

<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Linux-Event-Sources.png' alt="Linux - Event Sources" />

### Linux - Login Status

> See information about logins to Linux hosts; including logins by hour; failed logins per host; the top 30 successful and failed logins; and the top 30 successful and failed remote logins.

Reviewer: ```suggestion
The **Linux - Login Status** dashboard provides information about logins to Linux hosts; including logins by hour; failed logins per host; the top 30 successful and failed logins; and the top 30 successful and failed remote logins.
```

<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Linux-Login-Status.png' alt="Linux - Login Status" />

### Linux - Security Status

> See information about security on Linux hosts, including su, sudo attempts, new and existing user assignments, package operations, and system start events.

Reviewer: ```suggestion
The **Linux - Security Status** dashboard provides information about security on Linux hosts, including su, sudo attempts, new and existing user assignments, package operations, and system start events.
```
